test/conformance/api/v1/revision_timeout_test.go

```diff
@@ -206,7 +206,17 @@ func TestRevisionTimeout(t *testing.T) {
 		test.AddRootCAtoTransport(t.Logf, clients, test.ServingFlags.Https)); err != nil {
 		t.Fatalf("Error probing %s: %v", rev5sURL, err)
 	}
-
+	t.Log("Probing", rev2sURL)
+	if _, err := pkgTest.WaitForEndpointState(
+		clients.KubeClient,
+		t.Logf,
+		rev2sURL,
+		v1test.RetryingRouteInconsistency(pkgTest.IsOneOfStatusCodes(http.StatusOK, http.StatusGatewayTimeout)),
+		"WaitForSuccessfulResponse",
+		test.ServingFlags.ResolvableDomain,
+		test.AddRootCAtoTransport(t.Logf, clients, test.ServingFlags.Https)); err != nil {
+		t.Fatalf("Error probing %s: %v", rev2sURL, err)
```

Comment:
Why did we add those?

Reply:
We noticed that RevisionTimeout test fails with this change to activator because revision 2s version is spun down before assertions and the request goes through the activator. Following assertions failed constantly as request times out because time for pod to come up + delay added in test setup > revision timeout.
This ping ensures that both revisions are running before assertions and avoid the request path via activator.